<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edo_skinWeightIoUI</class>
 <widget class="QMainWindow" name="edo_skinWeightIoUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>edo_skinWeightIoUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="edo_skinWeightIoLayout">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>431</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="IO_skinweight">
     <attribute name="title">
      <string>IO_skinweight</string>
     </attribute>
     <widget class="QPushButton" name="ex_bt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>export skinWeight</string>
      </property>
     </widget>
     <widget class="QPushButton" name="md_bt">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>101</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>modify skinWeight</string>
      </property>
     </widget>
     <widget class="QPushButton" name="im_bt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>import skinWeight</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sw_line">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>79</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="rp_line">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SKIN WEIGHT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>REPLACE</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Edit_skinweight">
     <attribute name="title">
      <string>Edit_skinweight</string>
     </attribute>
     <widget class="QPushButton" name="saveTemp_bt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>FINISH</string>
      </property>
     </widget>
     <widget class="QListWidget" name="sourseInfluence">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>201</width>
        <height>181</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="directInfluence">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>201</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="direct_bt">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>LOAD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="source_bt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>LOAD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addTemp_bt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD TO TEMPLATE</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionHow_to_use"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHow_to_use">
   <property name="text">
    <string>how_to_use</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
